packages/bundler-plugin-core/src/utils/sentryUtils.ts

```diff
@@ -0,0 +1,120 @@
+import { type Options, type Output } from "src";
+import { green, red } from "./logging";
+export const FONT_FILE_EXTENSIONS = ["woff", "woff2", "ttf", "otf", "eot"];
+export const IMAGE_FILE_EXTENSIONS = [
+  "jpg",
+  "jpeg",
+  "png",
+  "gif",
+  "svg",
+  "webp",
+  "apng",
+  "avif",
+];
+interface SentryBundleStats {
+  total_size: number;
+  javascript_size: number;
+  css_size: number;
+  fonts_size: number;
+  images_size: number;
+  bundle_name: string;
+  environment: string;
+}
+
+export const sendSentryBundleStats = async (
+  output: Output,
+  userOptions: Options,
+) => {
+  const { sentryAuthToken, sentryOrganization, sentryProject } =
```

Comment:
We should try to grab the same environmental variables Sentry CLi looks for https://docs.sentry.io/product/cli/configuration/#configuration-values

Reply:
👍